Skype/UCWA/EntryExitAnnouncement_ref.md

```diff
@@ -1,4 +1,6 @@
-
+---
+title: EntryExitAnnouncement for attendance announcements in an online meeting
```

Comment:
Change to "EntryExitAnnouncement reference" (Most of the files in the UCWA and UCMA directories, if you just have the H1 text and the word "reference", you're golden.)

Reply:
done! thanks for the tips. :) changed both title and H1 title